storage/mongodb/files/scripts/test-mongo.sh

```diff
@@ -19,17 +21,33 @@ echo "NAMESPACE: ${NAMESPACE}"
 echo "ITERATION: ${ITERATION}"
 echo "THREADS: ${THREADS}"
 echo "WORKLOADS: ${WORKLOAD}"
+echo "RECORDCOUNT: ${RECORDCOUNT}"
+echo "OPERATIONCOUNT: ${OPERATIONCOUNT}"
 
-for i in $(seq 1 ${ITERATION});
-do
-  for load  in $(echo ${WORKLOAD} | sed -e s/,/" "/g);
-  do
-    echo "iteration: ${i} "and" ${load}"
-    ## TODO support to override other params
-    oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep mongodb | awk '{print $1}') -- scl enable rh-mongodb32 -- mongo -u redhat -p redhat ${MONGODB_IP}:27017/testdb --eval "db.usertable.remove({})"
-    oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep ycsb | awk '{print $1}') -- ./bin/ycsb load mongodb -s -threads "${THREADS}" -P "workloads/${load}" -p mongodb.url=mongodb://redhat:redhat@${MONGODB_IP}:27017/testdb > ${output_dir}/mongodb_${load}_result_run_${THREADS}_${i}.txt 2>&1
-    grep -E 'RunTime|Throughput' ${output_dir}/mongodb_${load}_result_run_${THREADS}_${i}.txt > ${output_dir}/mongodb_${load}_result_run_${THREADS}_${i}_brief.txt
-    grep Throughput ${output_dir}/mongodb_${load}_result_run_${THREADS}_${i}.txt | cut -d',' -f3 >> ${output_dir}/mongodb_${load}_result_run_${THREADS}_numbers.txt
-  done
+
+for i in $(seq 1 ${ITERATION}); do 
+  for load  in $(echo ${WORKLOAD} | sed -e s/,/" "/g); do 
+	for thread in $(echo ${THREADS} | sed -e s/,/" "/g); do 
+    		oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep mongodb | awk '{print $1}') -- scl enable rh-mongodb32 -- mongo -u redhat -p redhat ${MONGODB_IP}:27017/testdb --eval "db.usertable.remove({})"
+		oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep ycsb | awk '{print $1}') -- ./bin/ycsb load mongodb -s -threads $thread -P "workloads/${load}" -p mongodb.url=mongodb://redhat:redhat@${MONGODB_IP}:27017/testdb -p recordcount=${RECORDCOUNT} -p operationcount=${OPERATIONCOUNT} 2>&1 | tee -a ${output_dir}/mongodb_load_data_${load}_threads_${thread}.txt
+
+		oc -n ${NAMESPACE} exec $(oc get pod -n ${NAMESPACE} | grep ycsb | awk '{print $1}') -- ./bin/ycsb run mongodb -s -threads $thread -P "workloads/${load}" -p mongodb.url=mongodb://redhat:redhat@${MONGODB_IP}:27017/testdb 2>&1 -p recordcount=${RECORDCOUNT} -p operationcount=${OPERATIONCOUNT}  | tee -a ${output_dir}/mongodb_run_load_${load}_threads_${thread}.txt
+	done 
```

Comment:
Why do we run the same test twice here?

Reply:
NIP: for consistency ... 2>&1 after -p recordcount=${RECORDCOUNT} -p operationcount=${OPERATIONCOUNT}